server/igc/IgcAnalyzer.js

```diff
@@ -284,20 +284,32 @@ function stripByFactor(factor, input) {
 }
 
 function stripAroundTurnpoints(input, turnpoints) {
```

Comment:
Magst du bei Gelegenheit noch mal eine JSDoc hinzufügen, was genau die Funktion macht? Ich kann so auf den ersten Blick zb nicht erkennen, was `input` ist, oder warum die Funktion überhaupt gebraucht wird. Und müsste jetzt durch den restlichen igc analyser wühlen um nachzuvollziehen, was in der Änderung passiert.